src/components/Geo/index.jsx

```diff
@@ -15,20 +39,24 @@ const Geo = () => {
     return <div className="mx-4 my-2">
         {headTags}
         <div className="text-center text-xl">The Planetary RNA Virome</div>
-        <p className="py-1">
-            We searched 5.7 million public sequencing libraries for the RNA virus hallmark gene, RNA-dependent RNA Polymerase (RdRP).
-            This map shows the location of BioSamples from which an intact RdRP sequence could be recovered and geographical meta-data was present.
-            A 100-meter jitter is added to all points to prevent overplotting.
-        </p>
-        <p className="py-1">
-            Use the <b>Box Select</b> or <b>Lasso Select</b> icons in the top-right to retrieve a detailed sample list below the map.
-        </p>
-        <p className="py-1">
-            Currently in Beta Version. Feedback is welcome.
-        </p>
+
+
+        <div><Details title={helpIcon} defaultOpen={false}>
```

Comment:
![icons](https://user-images.githubusercontent.com/13424970/111093433-47c49000-850f-11eb-8754-388810544a75.png)

This seems a bit weird - too many icons. Suggestion: `> Details` or `> Info`

Reply:
:+1: